etc/bankdefs/hipo4/event-ai.json

```diff
@@ -154,6 +154,29 @@
             {"name":"layermult","type":"B", "info":"cluster layer multiplicity"}
         ]
     },
+    {
+        "name": "RECHBAI::CaloExtras",
+        "group": 400,
+        "item" : 24,
+        "info": "Calorimeter Cluster Parameters",
+        "entries": [
+            {"name":"dbstU",       "id":1, "type":"S", "info":"DB status + U strip*10"},
+            {"name":"dbstV",       "id":2, "type":"S", "info":"DB status + V strip*10"},
+            {"name":"dbstW",       "id":3, "type":"S", "info":"DB status + W strip*10"},
+            {"name":"rawEU",       "id":4, "type":"F", "info":"raw U peak energy"},
+            {"name":"rawEV",       "id":5, "type":"F", "info":"raw V peak energy"},
+            {"name":"rawEW",       "id":6, "type":"F", "info":"raw W peak energy"},
+            {"name":"recEU",       "id":7, "type":"F", "info":"recon U peak energy"},
+            {"name":"recEV",       "id":8, "type":"F", "info":"recon V peak energy"},
+            {"name":"recEW",       "id":9, "type":"F", "info":"recon W peak energy"},
+            {"name":"recDTU",      "id":10, "type":"F", "info":"recon U peak Dtime"},
```

Comment:
Same (minor) comment as I had on #966: it would be useful to change the Information field for the time variables to state explicitly "TDC-based time" and "fADC-based time" instead of Dtime and Ftime

Reply:
done, would be good if Cole confirmed the units